### PR TITLE
SAV-358: Facility info in central sync logging

### PR DESCRIPTION
**Pull request by mcccclean**
### Changes

Core change is to CentralSyncManager, just including a couple of data points necessary to get Honeycomb alerting us of failed syncs.

Also includes some updates to FacilitySyncManager, mostly just changing the format of the logs so that they are consistent with the terminology + format of the central-side ones.

### Checklist

- [x] Code is finished
- [ ] Tests are written
- [ ] UI Screenshots added to the Linear issue
- [ ] Testing notes added to the Linear issue

_Upon merging:_

- [ ] Update the changelog (find it [here](https://github.com/beyondessential/tamanu/releases) in the appropriate _draft_ release)
  - [ ] with a ticket reference (e.g. `TAN-123: a one line description`, keep the lists sorted)
  - [ ] any config/settings changes and manual deployment steps
  - [ ] any db schema or other changes that could impact downstream data analysis
- [ ] Update the [config reference](https://beyond-essential.slab.com/posts/reference-config-file-0c70ukly) or [settings reference](https://beyond-essential.slab.com/posts/reference-settings-0blw1x2q) as needed
- [ ] Update the [relevant runbook(s)](https://beyond-essential.slab.com/topics/runbooks-bs04ml6c) as needed

### CODE REVIEW

**Review by edmofro — CHANGES_REQUESTED** [review]
All lgtm, but notice that the auto-linter didn't work, so holding off approval.

Also created [this PR](https://github.com/beyondessential/tamanu/pull/4600) which would be a helpful addition to what you've got here. Haven't tested it yet and it is riskier than the changes currently in scope, so I'll leave it to you to decide whether it should be included

**Review by mcccclean — COMMENTED** [review]
Made a few changes since the last review as testing revealed that it wasn't successfully honeycombing the info

**Review by edmofro — APPROVED** [review]
lgtm

**Pull request merged**
